app/models/procedure_revision.rb

```diff
@@ -224,7 +224,7 @@ def coordinate_and_tdc(stable_id)
   end
 
   def routable_types_de_champ
-    types_de_champ_public.filter { |tdc| [:drop_down_list, :departements, :regions].include?(tdc.type_champ.to_sym) }
+    types_de_champ_public.filter { |tdc| [:drop_down_list, :departements, :regions, :communes].include?(tdc.type_champ.to_sym) }
```

Comment:
optionnel

- dans la classe TypeDeChamp
```
ROUTABLE_TYPES = ['drop_down_list', 'departements', 'regions', 'communes']
...
def routable? = type_champ.in?(ROUTABLE_TYPES)
```
- puis

```suggestion
    types_de_champ_public.filter { _1.routable? }
```